LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2020, conda-forge
+Copyright (c) 2020, conda-forge (@cconda-forge/core)
```

Comment:
isn't it `@conda-forge/core`... I think there are two 'c's at the start

Reply:
woops